package.json

```
@@ -4,7 +4,8 @@
   "main": "index.js",
   "license": "MIT",
   "scripts": {
-    "prettier": "prettier --write \"**/*.js\""
+    "prettier": "prettier --write \"**/*.js\"",
+    "lint": "./node_modules/.bin/eslint **/*.gs --quiet"
```

node_modules/.bin is added to env before running so this can just be the following. 
```suggestion
    "lint": "eslint **/*.js --quiet"
```